RadixWallet/Core/FeaturePrelude/AddressView/AddressFormat.swift

```diff
@@ -74,3 +74,16 @@ extension NonFungibleLocalId {
 		}
 	}
 }
+
+extension TXID {
```

Comment:
Aha! I suspected we had a regression, because I started adding this in Sargon for transaction hashes but then tested main branch and saw we did not truncated. I should have spoken up :) @micbakos-rdx we can you add a PR implementing this in Sargon?